### PR TITLE
Changed to use wharf-core/pkg/cacertutil instead of internal/httputils 

### PR DESCRIPTION
**Pull request by Alexamakans**
- \[x] I've added a new note in the `CHANGELOG.md` file, according to docs:
  https://iver-wharf.github.io/#/development/changelogs/writing-changelogs
- \[x] Wait for wharf-core release with change from https://github.com/iver-wharf/wharf-core/pull/27.

## Summary

- Changed to use `wharf-core/pkg/cacertutil` instead of `internal/httputils`
- Deleted `internal/httputils`

## Motivation

Copies of the `internal/httputils` package exists in multiple repos: [wharf-api](https://github.com/iver-wharf/wharf-api), [wharf-provider-github](https://github.com/iver-wharf/wharf-provider-github), [wharf-provider-gitlab](https://github.com/iver-wharf/wharf-provider-gitlab), and [wharf-provider-azuredevops](https://github.com/iver-wharf/wharf-provider-azuredevops).

Moving it to wharf-core reduces code duplication, and makes changing the code simpler.

## PR & Issue

Depends on pull request: https://github.com/iver-wharf/wharf-core/pull/27
Related issue: https://github.com/iver-wharf/wharf-core/issues/26

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,8 +5,8 @@ go 1.16
 require (
 	github.com/gin-contrib/cors v1.3.0
 	github.com/gin-gonic/gin v1.7.1
-	github.com/iver-wharf/wharf-api-client-go v1.4.0
```
Comment (applejag) [inline go.mod:8]: Why downgrade?

File: CHANGELOG.md [hunk 2]
```diff
@@ -69,6 +69,11 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Added support for the TZ environment variable (setting timezones ex.
   `"Europe/Stockholm"`) through the tzdata package. (#20)
 
+- Removed `internal/httputils`, which was moved to `wharf-core/pkg/cacertutil`.
+  (#28)
+
+- Added dependency on `wharf-core/pkg/cacertutil`. (#28)
```
Comment (applejag) [inline CHANGELOG.md]: nit: Missing full package name. And IMO you can omit these package dependencies. We already have a dedicated release note for the `github.com/iver-wharf/wharf-core` Go module.

**Review by applejag — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (Alexamakans) [reply go.mod:8]: I didn't see a changelog entry for it and thought I had accidentally upgraded it.
Updated to v1.4.0 again.

File: CHANGELOG.md (hunk 2 above)
Comment (Alexamakans) [reply CHANGELOG.md]: Removed `Added dependency` entry in 0de5fbc.

**Review by applejag — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 3]
```diff
@@ -69,6 +69,9 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Added support for the TZ environment variable (setting timezones ex.
   `"Europe/Stockholm"`) through the tzdata package. (#20)
 
+- Removed `internal/httputils`, which was moved to
+  `github.com/iver-wharf/wharf-core/pkg/cacertutil`. (#28)
+
```
Comment (applejag) [inline CHANGELOG.md]: v2.0.0 has been released. Remember to add a new WIP version section to the CHANGELOG.md

**Review by applejag — CHANGES_REQUESTED** [review]

**Review by fredx30 — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -12,13 +12,24 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
+## v2.1.0 (WIP)
```
Comment (applejag) [inline CHANGELOG.md]: nit: Think this is a patch update, and not a minor update.

No new features has been added.

**Review by applejag — CHANGES_REQUESTED** [review]

**Review by applejag — APPROVED** [review]
Meant to leave last review as approve. Misclick

File: CHANGELOG.md (hunk 4 above)
Comment (Alexamakans) [reply CHANGELOG.md:15]:
```suggestion
## v2.0.2 (WIP)
```

Comment (Alexamakans) [reply CHANGELOG.md]:
```suggestion
## v2.0.2 (WIP)
```

**Pull request merged**